### PR TITLE
Add specs for configuration through imports

### PR DESCRIPTION
**Pull request by jathak**
See sass/sass#2772

[skip dart-sass]

### CODE REVIEW

**Comment by jathak** [comment]
@nex3: What additional specs should be added for this?

File: spec/directives/import/configuration.hrx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  c: configured;
+}
+
+b {
+  c: configured;
+}
+
+<===>
+================================================================================
+<===> import-twice/with-change/input.scss
```
Comment (nex3) [inline spec/directives/import/configuration.hrx]: I think you also want a test that verifies that, after the second import, the variable does have the new value even though the module hasn't been reloaded.

**Comment by nex3** [comment]
It just occurred to me that you should probably also add a spec for a local variable set for a nested import.

**Comment by nex3** [comment]
Also, add a spec that verifies no error is thrown if a variable is set in the import context that *isn't* defined in the forwarded module.

**Comment by nex3** [comment]
And based on our conversation in the implementation CL:

* A spec that verifies that, if the midstream file overrides the input's definition of the variable, the input's definition is still used in the configuration.
* A spec that verifies that, if the midstream file defines a new variable with the same name as a configurable variable, it doesn't get included in the config.

**Comment by jathak** [comment]
Okay, I think I added all the tests you suggested. The one for confirming that a declaration between duplicate `@import` rules still sets it in the current file is todo for now because of sass/dart-sass#888.

**Pull request merged**